source/Integrate/Partners/account-migration.md

```diff
@@ -0,0 +1,54 @@
+---
+seo:
+  title: SendGrid account migration
+  description: Sometimes if a SendGrid partner no longer supports SendGrid core functionality, you may want to migrate your account to a regular SendGrid account.
+  keywords: migrate, partner sunset
+title: SendGrid account migration
+weight: 0
+layout: page
+navigation:
+  show: true
+---
+
+If you are migrating from one SendGrid account to another, there are several things to keep in mind - make sure to stop sending mail until you migrate to a new, warmed up account, and make sure export data from your old account, and clean up the data described here so that you can import it into your new account.
+
+Before you begin
+
+- Pause sending email on your old account.
+- Login to app.sendgrid.com with your current SendGrid username and password.
+
+1. **Sender authentication** You cannot export and import sender authentication data - you need to set this up manually in your new account after you delete the current authentication records from your [DNS]({{root_url}}/User_Guide/Settings/Sender_authentication/How_to_set_up_domain_authentication.html#-DNS) provider.
+
+If you have domain, link, and IP authentication set up, remove 5 CNAME or MX/TXT records and 1 A record from your domain’s DNS. To verify the records delete from your DNS, go to the [Sender Authentication page](https://app.sendgrid.com/settings/sender_auth) in the UI and check the DNS column:
+
+![]({{root_url}}/images/dns-records.png "Where to find your existing DNS records")
+
+To set up sender authentication in your new account, check out [Setting up domain authentication]({{root_url}}/User_Guide/Settings/Sender_authentication/How_to_set_up_domain_authentication.html), [Setting up reverse DNS]({{root_url}}/User_Guide/Settings/Sender_authentication/How_to_set_up_link_branding.html), and [Setting up link branding]({{root_url}}/User_Guide/Settings/Sender_authentication/How_to_set_up_link_branding.html).
+
+2. **Migrate Unsubscribes** - To download your unsubscribes, on the [Global unsubscribes page](https://app.sendgrid.com/suppressions/global_unsubscribes), select the action cog, and then click *Export CSV*.
```

Comment:
That damn action cog....😝I checked the UI and it says "Download as CSV" on the Global unsubscribes page. (Which is, of course, different from what it says on the unsubscribe groups page, but what you gonna do?)